### ISSUE TITLE
404 doc to english doc

### ISSUE THREAD

**Issue by vansin** [open-mmlab/mmengine#666]
能不能干脆把所有还没指定跳转的 有跳转功能的（比如某个页面切换英文文档都重定向到英文文档的开头 省事也避免报错)

Can you simply all have not specified the jump jump function (such as a page switch to the English document are redirected to the beginning of the English document to save time and avoid reporting errors)

**Comment by sanbuphy** [comment]
Look forward to this update

**Comment by C1rN09** [comment]
Hi, thanks for your report! English documentations are currently under translation and will be released about 2 weeks later. Stay tuned!

As for your suggestion, I don't quite catch the point... Can you give an example? Maybe you can also post a PR :smile_cat:

**Comment by sanbuphy** [comment]
> 

If I  switch the language  in docs(click `Read the Docs` to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404

**Comment by sanbuphy** [comment]
> If I switch the language in docs(click `Read the Docs` to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404

If I switch the language in docs(click Read the Docs to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404

**Comment by C1rN09** [comment]
> > If I switch the language in docs(click `Read the Docs` to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404
> 
> If I switch the language in docs(click Read the Docs to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404

Can you provide more information on this, such as which page fails switching language or which button you clicked? I tried several pages and buttons(show as below) but failed to reproduce.

![button1](https://user-images.githubusercontent.com/112053249/199703849-00ab2726-8926-479e-aeba-515ca44be4a1.png)
![button2](https://user-images.githubusercontent.com/112053249/199704003-35851ea9-c067-4419-b77b-c59e342c0b9f.png)
![button3](https://user-images.githubusercontent.com/112053249/199704087-3e67fdec-873e-45d3-beb8-a3fe08d23a84.png)

**Comment by sanbuphy** [comment]
> > > If I switch the language in docs(click `Read the Docs` to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404
> > 
> > 
> > If I switch the language in docs(click Read the Docs to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404
> 
> Can you provide more information on this, such as which page fails switching language or which button you clicked? I tried several pages and buttons(show as below) but failed to reproduce.
> 
> ![button1](https://user-images.githubusercontent.com/112053249/199703849-00ab2726-8926-479e-aeba-515ca44be4a1.png) ![button2](https://user-images.githubusercontent.com/112053249/199704003-35851ea9-c067-4419-b77b-c59e342c0b9f.png) ![button3](https://user-images.githubusercontent.com/112053249/199704087-3e67fdec-873e-45d3-beb8-a3fe08d23a84.png)

Yes,you are right. I think it works well in MMEngine,but if you try it in MMDet documentation, you will meet the wrong

**Comment by sanbuphy** [comment]
> > > If I switch the language in docs(click `Read the Docs` to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404
> > 
> > 
> > If I switch the language in docs(click Read the Docs to switch), I will meet 404. I hope it will redirected to the begin of docs avoid 404
> 
> Can you provide more information on this, such as which page fails switching language or which button you clicked? I tried several pages and buttons(show as below) but failed to reproduce.
> 
> ![button1](https://user-images.githubusercontent.com/112053249/199703849-00ab2726-8926-479e-aeba-515ca44be4a1.png) ![button2](https://user-images.githubusercontent.com/112053249/199704003-35851ea9-c067-4419-b77b-c59e342c0b9f.png) ![button3](https://user-images.githubusercontent.com/112053249/199704087-3e67fdec-873e-45d3-beb8-a3fe08d23a84.png)

like this : https://mmdetection.readthedocs.io/zh_CN/3.x/advanced_guides/index.html#id2
if you  use `read the docs` to change language,you can see  404 error page.

**Comment by C1rN09** [comment]
I think it might be an issue in MMDetection. Maybe we should invite @ZwwWayne to have a look

**Comment by sanbuphy** [comment]
> I think it might be an issue in MMDetection. Maybe we should invite @ZwwWayne to have a look

Yes I have created a issue in MMDetection, Thank you.
